web_programming/emails_from_url_alt.py

```diff
@@ -88,6 +87,8 @@ def getmails(num: int) -> None:
             print(mail)
         sys.exit(1)
 
+    return emails
+
 
 if __name__ == "__main__":
     user_url = "https://google.com"  # Example Url
```

Comment:
Let's just replace ___all___ of `__main__` with:
```suggestion
if __name__ == "__main__":
    print("\n".join(getmails("https://google.com")))
```

Reply:
It takes `num` as argument. I tried to add url too , but didn't work the way i want, and also `popleft` is faster then `pop(0)`  (if using list). So set would be faster.